:visited: edits

apastan opened the pull request:
### Description

1. It would be good to note that the pseudo-class `:visited` applies only to elements `<a>` and `<area>` with `href` attribute. Source - [HTML Living Standard](https://html.spec.whatwg.org/multipage/semantics-other.html#selector-visited)

2.  The `:visited` pseudo-class and `:link` pseudo-class are mutually exclusive.  Source - [Selectors Level 4
W3C Working Draft](https://www.w3.org/TR/selectors-4/#link)

3.  <blockquote>Styles defined by the :visited pseudo-class will be overridden by any subsequent link-related pseudo-class (:link, :hover, or :active) that has at least equal specificity.</blockquote>

":visited" pseudo-class can't be overridden by ":link" pseudo-class because they are mutually exclusive. [Example](https://jsfiddle.net/s9dyctq6/3/)

```
a:visited {
/* The visited link still will be "rebeccapurple" color despite of "a:link" declaration comes after "a:link" declaration. */
  color: rebeccapurple;
}

a:link {
  color: red;
}
```

apastan committed:
Update index.md

1. It would be good to note that the pseudo-class ":visited" applies only to elements <a> and <area> with href attribute.
https://html.spec.whatwg.org/multipage/semantics-other.html#selector-visited

2.  The `:visited` pseudo-class and `:link` pseudo-class are mutually exclusive. 
https://www.w3.org/TR/selectors-4/#link

3.  "Styles defined by the :visited pseudo-class will be overridden by any subsequent link-related pseudo-class (:link, :hover, or :active) that has at least equal specificity."

":visited" pseudo-class can't be overridden by ":link" pseudo-class because they are mutually exclusive.
The visited link still will be "rebeccapurple" color despite of "a:link" declaration comes after "a:link" declaration.

      a:visited {
        color: rebeccapurple;
      }

      a:link {
        color: red;
      }

apastan committed:
Update index.md

apastan committed:
Merge branch 'main' into patch-1

estelle commented on files/en-us/web/css/_colon_visited/index.md:
```diff
@@ -14,11 +14,11 @@ browser-compat: css.selectors.visited
 {{CSSRef}}
 
-The **`:visited`** [CSS](/en-US/docs/Web/CSS) [pseudo-class](/en-US/docs/Web/CSS/Pseudo-classes) represents links that the user has already visited. For privacy reasons, the styles that can be modified using this selector are very limited.
+The **`:visited`** [CSS](/en-US/docs/Web/CSS) [pseudo-class](/en-US/docs/Web/CSS/Pseudo-classes) applies once the link has been visited by the user. For privacy reasons, the styles that can be modified using this selector are very limited. The `:visited` pseudo-class applies only {{htmlelement("a")}} and {{htmlelement("area")}} elements that have an `href` attribute. The `:visited` pseudo-class a ...
```
```suggestion
The **`:visited`** [CSS](/en-US/docs/Web/CSS) [pseudo-class](/en-US/docs/Web/CSS/Pseudo-classes) applies once the link has been visited by the user. For privacy reasons, the styles that can be modified using this selector are very limited. The `:visited` pseudo-class applies only {{htmlelement("a")}} and {{htmlelement("area")}} elements that have an `href` attribute. 
```

let's put it lt later. The first paragraph is a summary statement used by other resources such at developer tools. So we limit it to what the feature is. What it isn't can come later.

estelle commented on files/en-us/web/css/_colon_visited/index.md:
```diff
@@ -14,11 +14,11 @@ browser-compat: css.selectors.visited
 {{EmbedInteractiveExample("pages/tabbed/pseudo-class-visited.html", "tabbed-shorter")}}
 
-Styles defined by the `:visited` pseudo-class will be overridden by any subsequent link-related pseudo-class ({{cssxref(":link")}}, {{cssxref(":hover")}}, or {{cssxref(":active")}}) that has at least equal specificity. To style links appropriately, put the `:visited` rule after the `:link` rule but before the `:hover` and `:active` rules, as defined by the _LVHA-order_: `:link` — `:visited` — `:h ...
+Styles defined by the `:visited` and `:link` pseudo-classes can be overridden by user-action pseudo-classes ({{cssxref(":hover")}} or {{cssxref(":active")}}) that have at least equal specificity and come after `:visited` and `:link` pseudo-classes. To style links appropriately, put the `:visited` rule after the `:link` rule but before the `:hover` and `:active` rules, as defined by the _LVHA-orde ...
```
```suggestion
Styles defined by the `:visited` and unvisited [`:link`](/en-US/docs/Web/CSS/:link) pseudo-classes can be overridden by user-action pseudo-classes ({{cssxref(":hover")}} or {{cssxref(":active")}}) that have at least equal specificity and come after `:visited` and `:link` pseudo-classes. To style links appropriately, put the `:visited` rule after the `:link` rule but before the `:hover` and `:active` rules, as defined by the _LVHA-order_: `:link` — `:visited` — `:hover` — `:active`. The `:visited` pseudo-class and `:link` pseudo-class are mutually exclusive.
```

estelle commented on files/en-us/web/css/_colon_visited/index.md:
````diff
@@ -35,7 +35,7 @@ For privacy reasons, browsers strictly limit which styles you can apply using th
 
 ```
-visited
+selector:visited
````
```suggestion
:visited
```

estelle reviewed (commented):
a few suggested edits.

apastan committed:
:visited: definition edits

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

apastan committed:
:visited: syntax edits

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

apastan committed:
:visited: edits

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

apastan committed:
Merge branch 'main' into patch-1

apastan commented:
Thanks for your suggestions. Please, let me know if I need to do something.

apastan committed:
Merge branch 'main' into patch-1

apastan committed:
:visited: final edits

Make the second paragraph more similar to the current version of the page. Now it is a little shorter.

estelle reviewed (approved):
thank you
